lib/coins/btc.js

```diff
@@ -34,23 +35,53 @@ function formatAddress (address) {
   return address
 }
 
-function validate (network, address) {
+function base58Validator (network, address) {
   try {
-    if (!network) throw new Error('No network supplied.')
-    if (!address) throw new Error('No address supplied.')
-
     const buf = bs58check.decode(address)
     const addressType = buf[0]
 
-    if (buf.length !== 21) throw new Error(`Invalid length: ${buf.length}`)
+    if (buf.length !== 21) {
+      console.log(`Invalid base58 address length: ${buf.length}`)
+    }
```

Comment:
Shouldn't we return ``false`` here?

```suggestion
      return false      
    }
```